nbproject/ant-deploy.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We do not need anything in the nbproject directory. You deleted that line above.